apis/v1alpha1/tcproute_types.go

```diff
@@ -51,15 +54,18 @@ type TCPRouteStatus struct {
 // TCPRouteRule is the configuration for a given rule.
 type TCPRouteRule struct {
 	// Matches define conditions used for matching the rule against
-	// incoming TCP connections.
-	// Each match is independent, i.e. this rule will be matched
-	// if **any** one of the matches is satisfied.
+	// incoming TCP connections. Each match is independent, i.e. this
+	// rule will be matched if **any** one of the matches is satisfied.
+	// If unspecified, all requests from the associated gateway TCP
+	// listener will match.
 	//
 	// +optional
 	// +kubebuilder:validation:MaxItems=8
 	Matches []TCPRouteMatch `json:"matches,omitempty"`
 
-	// ForwardTo defines the backend(s) where matching requests should be sent.
+	// ForwardTo defines the backend(s) where matching requests should be
+	// sent. If unspecified, matching traffic will not be forwarded.
```

Comment:
This is more of a general behavior question (not specific to TCP):
What should be the behavior in this case?
Should the request instead be sent to the forwarding target of the next match (in priority order)?
Or should the request be dropped i.e. 5xx or similar?

Reply:
The language I propose above is consistent with `HTTPRoute`. I can see a potential use case to expose a "honeypot" endpoint where the matched requests get dropped.